parsec/backend/postgresql/organization.py

```diff
@@ -159,6 +161,67 @@
 """
 )
 
+_q_get_server_stats = Q(
```

Comment:
This seems pretty complex, do you think we could instead:
- modify the organization_stats to include the `from_date`/`to_date` parameters
- implement server_stats as a call to retrieve the list of organizations, then calling organization_stats on each one of them

This would increase a bit the complexity of organization_stats (my hope is we would only have to add date check conditions on where clauses in the items of `_q_get_stats`), but it would drastically simplify server_stats (both in the postgresql and memory implementations \o/)

Aaaand this means `ServerStats` would basically be a `Dict[OrganizationID, OrganizationStats]`